### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by pali** [ladislav-zezula/StormLib#93]
Hi! In git are couple of fixes since last version was released. When is planned new StormLib release?

**Comment by ladislav-zezula** [comment]
Yeah, could do. As soon as I have time.

**Comment by ladislav-zezula** [comment]
Done.

**Comment by cen1** [comment]
Just a small note for future releases: please also lift the 
```
SET(VERSION_MAJOR "9")
SET(VERSION_MINOR "21")
SET(VERSION_PATCH "0")
```
in CMakeLists.txt for the next release. I use these versions when packaging.

**Comment by ladislav-zezula** [comment]
Fixed in commit e9f9b9bc58c63a72285920b889c8d434e74e8a28.
Thanks for the report.
